### PR TITLE
[CNTO-8437] - Plivo cx changes

### PR DESCRIPTION
**Pull request by esrathkkm**

### CODE REVIEW

File: app/public/manifest.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "Contacto",
-  "version": "0.0.3",
+  "name": "Plivo CX Service",
+  "version": "1.0.4",
```
Comment (mimanshi-plivo) [inline app/public/manifest.json]: shouldn't it be 1.0.0?

**Review by mimanshi-plivo — CHANGES_REQUESTED** [review]

File: app/public/manifest.json (hunk 1 above)
Comment (samuellawerentz) [reply app/public/manifest.json]: The released version is 1.0.3 somehow and we don't have a track of that in main branch. It was 2 years back and I published it locally and missed to update the version I guess.

**Review by samuellawerentz — APPROVED** [review]
Can we add some screenshots? I've approved the PR anyways.

File: app/public/manifest.json (hunk 1 above)
Comment (esrathkkm) [reply app/public/manifest.json]: there is already a version of 1.0.3 in chrome webstore. That update is not pushed to main branch somehow. So updated to 1.0.4

**Comment by esrathkkm** [comment]
Screenshot 2024-03-13 at 6 05 07 PM

Screenshot 2024-03-13 at 6 06 18 PM

Screenshot 2024-03-13 at 6 05 00 PM

@samuellawerentz

File: app/public/manifest.json (hunk 1 above)
Comment (mimanshi-plivo) [reply app/public/manifest.json]: okay

**Review by mimanshi-plivo — APPROVED** [review]

**Pull request merged**